src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Container } from "@mui/material";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { FooterNav } from "~/components/Footer/Nav.tsx";
+import { SocialIcons } from "~/components/Footer/social-icons.tsx";
+import {
+	FooterBottomContainer,
+	FooterContainer,
+	SocialIconsContainer,
+	StyledDivider,
+	StyledFooterCopyrightText,
+	StyledFooterText,
+	StyledFooterTextContainer,
+} from "~/components/Footer/styles.ts";
+import { MainLogo } from "~/components/Header/main-logo.tsx";
+import { colors } from "~/libs/constants/color.ts";
+import { footerColumnsLogIn } from "~/libs/constants/footer-links.ts";
+import {
+	socialMediaDataLogIn,
+	socialMediaDataLogOut,
+} from "~/libs/constants/social-media-icons.ts";
+import { useAppSelector } from "~/redux/hooks/use-app-selector.ts";
+import { RootState } from "~/redux/store.ts";
+
+export const Footer: React.FC = () => {
+	const { t } = useTranslation();
+	const isLoggIned = Boolean(
+		useAppSelector((state: RootState) => state.auth.accessToken),
+	);
+	const footerBgColor = isLoggIned ? colors.GREYRISH_RED : colors.CREME;
```

Comment:
the empty line above. please do not make one text from code, its difficult to read even book has paragraphs